test/powershell/Language/Parser/Parsing.Tests.ps1

```diff
@@ -665,6 +665,14 @@ Describe "Parsing using statement with alias and linebreak and comma" -Tag CI {
     }
 }
 
+It "Should correctly parse array literals for index expressions in method calls" {
+    $tks = $null
+    $ers = $null
+    $Script = '[string]::join(" ", (0, 1, 2)[0, 1])'
+    $result = [System.Management.Automation.Language.Parser]::ParseInput($Script, [ref]$tks, [ref]$ers)
+    $result.EndBlock.Statements[0].PipelineElements[0].Expression.Arguments[1].Index.Elements.Count | Should -Be 2
```

Comment:
you should also probably check to be sure that `$ers` has no contents (as it certainly does now)